go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -238,32 +255,29 @@ func (c *Controller) syncServices(key string) error {
 	if err != nil && !apierrors.IsNotFound(err) {
 		return err
 	}
-	// Get current state of the Service from the Service tracker
-	// These are the VIPs (ClusterIP:Port) that we have seen so far
-	// If the Service has updated the VIPs (has changed the Ports)
-	// and some were removed we have to delete those
-	// We need to create a map to not mutate the service tracker VIPs
-	vipsTracked := sets.NewString().Union(c.serviceTracker.getService(name, namespace))
+
 	// Delete the Service VIPs from OVN if:
```

Comment:
This should now read "Delete the Service's LB from OVN if:

Reply:
yup